### PR TITLE
Fix Duplicate GUID

### LINKED ISSUE #1357: Duplicate GUID
If a client crashes or has connection issues resulting in reconnecting to the server, it happens that connecting is denied with the message.
`Duplicate GUID`

Witch indicates that the server thinks the unresponsive but still connected client is not lagging. 

https://github.com/Unvanquished/Unvanquished/blob/2783e77acc2e41fc1f717124394ed851f1b612e6/src/sgame/sg_client.cpp#L1218-L1222

### PR DESCRIPTION
**Pull request by Gireen**
count a ping of 0 also as lagging
fix #1357

### CODE REVIEW

File: src/sgame/sg_utils.cpp [hunk 1]
```diff
@@ -451,7 +451,7 @@ bool G_ClientIsLagging( gclient_t *client )
 {
 	if ( client )
 	{
-		if ( client->ps.ping >= 999 )
```
Comment (illwieckz) [inline src/sgame/sg_utils.cpp:454]: How does it work with bots?

**Comment by illwieckz** [comment]
0.52 will be our best release ever.

File: src/sgame/sg_utils.cpp (hunk 1 above)
Comment (Gireen) [reply src/sgame/sg_utils.cpp:454]: this function is only once used.
when a client with duplicate guid connects.

there is probably a problem in a different place that lagging clients have a ping of 0 but this solves the problem

File: src/sgame/sg_client.cpp [hunk 2]
```diff
@@ -1217,7 +1217,12 @@ const char *ClientConnect( int clientNum, bool firstTime )
 		{
 			if ( !G_ClientIsLagging( level.clients + i ) )
 			{
+				G_LogPrintf( "Duplicate GUID: %i", i );
 				trap_SendServerCommand( i, "cp \"Your GUID is not secure\"" );
+				G_LogPrintf( "FailedConnecting: %i [%s] (%s) \"%s^*\" \"%s^*\"",
+				             clientNum, client->pers.ip.str[0] ? client->pers.ip.str : "127.0.0.1", client->pers.guid,
+				             client->pers.netname,
```
[the remark this answers is not in the document]
Comment (Gireen) [reply src/sgame/sg_client.cpp:1224]: The first one is the id of the connected client. The second informations about the connecting client, witch could also be an impostor.

I could combine both into one single message if there is a problem with it.

[the remark this answers is not in the document]
Comment (slipher) [reply src/sgame/sg_client.cpp:1224]: The exact same expression `client->pers.netname` is passed twice in a row.

[the remark this answers is not in the document]
Comment (Gireen) [reply src/sgame/sg_client.cpp:1224]: Oh that. Its the same as normal connecting  log messages.
https://github.com/Unvanquished/Unvanquished/blob/db422e82b7751a56e9df984904935cdd5b9dfd63/src/sgame/sg_client.cpp#L1247-L1250

One should be with and one without colors.

[the remark this answers is not in the document]
Comment (illwieckz) [reply src/sgame/sg_client.cpp:1224]: At least this is consistent with the others. I opened #1378 for related question of stripping color codes in the second one, if required, fixes for the others can be applied to this as well.

**Pull request merged**